tests/test_minhash.py

```diff
@@ -559,6 +559,23 @@ def test_similarity_1(track_abundance):
     assert round(b.similarity(b), 3) == 1.0
 
 
+def test_copy(track_abundance):
+    a = MinHash(20, 21, track_abundance=track_abundance)
+    a.add_hash(5)
+    a.add_hash(6)
+    b = a.copy()
+    assert a == b
```

Comment:
great start!

to add to this test, would be good to add another hash to either a or b, and then verify that they are NOT equal any more.